container_pipeline/workers/build.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+import hashlib
+import json
+import logging
+import os
+
+from container_pipeline.utils import Build, get_job_name
+from container_pipeline.lib.logging import load_logger
+from .base import BaseWorker
+
+
+class BuildWorker(BaseWorker):
+    """Build worker"""
+
+    def handle_job(self, job):
+        """Handle job"""
+        parent_build_running = False
+        parents = job.get('depends_on', '').split(',')
```

Comment:
In the container-index CI, we are having a check to ensure that `depends-on` isn't empty, right? @mohammedzee1000